### ISSUE TITLE
Visual que on Themes screen visually showing which themes are auto updated.

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/wp-autoupdates#48]
The themes screen.

![Screen Shot 2020-04-01 at 21 31 50](https://user-images.githubusercontent.com/5323259/78178632-3a2e0b80-7460-11ea-9d77-7ebb659b4122.png)

One can not at a quick glance know which themes are being autoupdated.

Clicking a theme:
![Screen Shot 2020-04-01 at 21 32 53](https://user-images.githubusercontent.com/5323259/78178723-621d6f00-7460-11ea-83c8-c250d3703589.png)

One sees the icon and message "Disable automatic updates"

Adding the icon to the themes page could look something like this:

![Themes-Auto-Updates-visual-que-icon](https://user-images.githubusercontent.com/5323259/78179205-1dde9e80-7461-11ea-9876-bc254f98b183.jpg)

At a quick glance one can see which themes are being Auto updated.

**Comment by hedgefield** [comment]
Yes, something like that would be quite nice. If overlaid on the image, it will need a good contrasting background to stand out. I can also see it in the bottom bar behind the theme name, might be easier to spot. Just gotta make sure it stays visible if the theme name is longer than the available space.

It's unfortunate that the Themes page has so little affordance for doing quick actions like the plugins page does, but that's a general WP gripe I have ;)

**Comment by paaljoachim** [comment]
Hei Tim

Here is another variation.

![Themes-screen-Auto-update-icon](https://user-images.githubusercontent.com/5323259/78413615-71471d00-7618-11ea-8267-bb7b61e1b5c8.jpg)

I also made another issue where I am doing even more experimentation https://github.com/WordPress/wp-autoupdates/issues/58. 
We should bring it up during next weeks design meeting.

**Comment by paaljoachim** [comment]
Here is a wireframe that removes the icons for themes that are not auto updated.

![Themes-screen-Auto-Updates-2](https://user-images.githubusercontent.com/5323259/79256424-4b2b4380-7e88-11ea-990c-127663d057db.jpg)

As it just shows a cycle icon for the themes where Auto update is activated.

**Comment by paaljoachim** [comment]
Here are three wireframes of the themes screen.
(Made in Affinity Photo.)

Example 1.
![Auto-updates-themes-screen](https://user-images.githubusercontent.com/5323259/79446243-352f9700-7fde-11ea-83b1-0ddf057486fd.jpg)

---

Example 2.
![Auto-updates-themes-screen-2](https://user-images.githubusercontent.com/5323259/79446303-4d071b00-7fde-11ea-9726-f6a6b3ed7790.jpg)

---

Example 3.
![Auto-updates-themes-screen-3](https://user-images.githubusercontent.com/5323259/79447158-bb001200-7fdf-11ea-8acf-45592fe859a6.jpg)




@audrasjb @pbiron

**Comment by afragen** [comment]
@paaljoachim currently the update notification is at the top of the theme card not the bottom. It's probably best to continue with that placement.

**Comment by wittwitsan** [comment]
I did a mockup for Themes page, adding more container for auto-updates before theme-id-container.

![Auto-updates_Themes_2020-04-22](https://user-images.githubusercontent.com/426383/79893069-b5665a00-842d-11ea-90b0-838b5dd31df8.png)

**Comment by abuyoyo** [comment]
+1 for the slimmer blocks by @ibdz 
I'd prefer the auto-update blocks to be _under_ the theme title/Active/Customize block.
And I think the upgrade icons (blue, orange, strike-through, etc.) should be incorporated into these blocks.

Personally I don't see any importance for the update notification to appear at the top of the card (as is the current design).
